crates/bevy_core_pipeline/src/core_3d/mod.rs

```diff
@@ -149,6 +151,41 @@ impl CachedRenderPipelinePhaseItem for AlphaMask3d {
     }
 }
 
+pub struct HashedAlpha3d {
```

Comment:
Docs :)

Reply:
@alice-i-cecile Would you like me to add docs to the other pipelines while I'm at it?

Reply:
Yes please! Rendering badly needs more docs, and I don't mind splashing those in. Make sure to do it in a clean commit though, in case we need to cherrypick.

Reply:
Ok